nextcloud-aio-helm-chart/readme.md

```diff
@@ -1,3 +1,10 @@
 # You can also install the AIO containers on Kubernetes using this Helm Chart
 
 This is currently beta and not ready yet.
+
+## How to use this?
+
+```
+helm repo add nextcloud-aio https://nextcloud.github.io/all-in-one/
+helm install my-release nextcloud-aio/nextcloud-aio-helm-chart
```

Comment:
Need to add a hint to overwrite values from https://github.com/nextcloud/all-in-one/blob/main/nextcloud-aio-helm-chart/values.yaml. Especially at least the ones marked with todo

Reply:
`-f my-values.yaml`

Reply:
done